lib/pages/setting/widgets/avatar.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:structure_flutter/core/resource/assets_images.dart';
+import '../../../core/resource/text_style.dart';
+
+Widget avatar(User user) {
```

Comment:
In my opinion, when u extract a widget to a class, u should use class widget (Stateless widget) instead of functional widgets
☛ https://github.com/flutter/flutter/issues/19269
☛ https://stackoverflow.com/questions/53234825/what-is-the-difference-between-functions-and-classes-to-create-reusable-widgets